eox_core/api/v1/views.py

```diff
@@ -200,6 +211,93 @@ def patch(self, request, *args, **kwargs):
         return Response(serialized_user.data)
 
 
+class EdxappRemoveUser(UserQueryMixin, APIView):
+    """
+    Handles API requests to remove an edxapp User.
+    """
+
+    authentication_classes = (BearerAuthentication, SessionAuthentication)
+    permission_classes = (EoxCoreAPIPermission,)
+    renderer_classes = (JSONRenderer, BrowsableAPIRenderer)
+
+    def delete(self, request, *args, **kwargs):  # pylint: disable=too-many-locals
+        """
+        Allows to safely remove an edxapp User.
+
+        For example:
+
+        **Requests**:
+            DELETE <domain>/eox-core/api/v1/remove-user/
```

Comment:
Why don't we use the existing `/eox-core/api/v1/user/`?

Reply:
@MoisesGSalas I was unsure about where to put it. I thought that maybe later we would want to have exclusive permissions for these endpoints that are intended to be used on the CLI, that's why I put it in a different view

Reply:
In that case what about an endpoint for CLI usage? maybe we can consider this a different API as it's meant to be used by the support team?

Reply:
@MoisesGSalas done, I added the support API